packages/expo-dev-launcher/bundle/functions/getInitialData.ts

```diff
@@ -21,9 +24,14 @@ export async function getInitialData(): Promise<Partial<AppProvidersProps>> {
 
   const initialBuildInfo = await getBuildInfoAsync();
   const initialDevMenuPreferences = await getMenuPreferencesAsync();
-
   const initialCrashReport = await getCrashReport();
 
+  if (isAuthenticated && updatesConfig.usesEASUpdates) {
+    prefetchBranchesForApp(updatesConfig.appId, updatesConfig.runtimeVersion).catch((error) =>
+      console.log({ error })
```

Comment:
Are we going to do something about this error?

Reply:
if this fails for whatever reason, we'll still try to fetch the branches when the extensions panel mounts, so I don't think there's a reason to do anything